src/libraries/Microsoft.PowerFx.Interpreter/Functions/Library.cs

```diff
@@ -34,8 +34,26 @@ private static T GetService<T>(this IServiceProvider services, T defaultService)
 
         public static IEnumerable<TexlFunction> FunctionList => FuncsByName.Keys;
 
+        public static readonly IReadOnlyDictionary<TexlFunction, AsyncFunctionPtr> FuncsByName;
+
+        static Library()
+        {
+            var funcsByName = new Dictionary<TexlFunction, AsyncFunctionPtr>();
+            foreach (var func in SimpleFuncsByName)
+            {
+                funcsByName.Add(func.Key, func.Value);
```

Comment:
>func.Key [](http://example.com/codeflow?start=32&length=8)

This could be an issue for overloads / name conflicts.

Reply:
The name of this field is misleading - it's not based on name, but based on the function implementation itself. But since this is a public field, it's possible that someone has a dependency on it, so I don't want to change it right now.

Reply:
Ah, just saw your other comment; will update it in this PR then.